codex/manifest/coders.nim

```diff
@@ -158,8 +158,8 @@ func decode*(_: DagPBCoder, data: openArray[byte]): ?!Manifest =
       protected: pbErasureInfo.buffer.len > 0)
 
   if self.protected:
-    self.K = K.int
-    self.M = M.int
+    self.ecK = ecK.int
+    self.ecM= ecM.int
```

Comment:
missing space

Reply:
I wonder why this wasn't caught by the style check. It would be very nice if we could convince it to...